website/content/docs/connect/cluster-peering/k8s.mdx

```diff
@@ -405,12 +405,26 @@ The examples described in this section demonstrate how to export a service named
 ## End a peering connection
 
 To end a peering connection, delete both the `PeeringAcceptor` and `PeeringDialer` resources.
+ 
+1. Delete the `PeeringDialer` resource from the second cluster.
 
-To confirm that you deleted your peering connection, in `cluster-01`, query the `/health` HTTP endpoint. The peered services should no longer appear.
+  ```shell-session
+  $ kubectl --context $CLUSTER2_CONTEXT delete --filename dialer.yaml 
+  ```
 
-```shell-session
-$ curl "localhost:8500/v1/health/connect/backend?peer=cluster-02"
-```
+1. Delete the `PeeringAcceptor` resource from the first cluster.
+
+  ```shell-session
+  $ kubectl --context $CLUSTER1_CONTEXT delete --filename acceptor.yaml
+  ````
+
+1. Confirm that you deleted your peering connection, in `cluster-01`, by querying the the `/health` HTTP endpoint. The peered services should no longer appear.
+
+  ```shell-session
+  $ kubectl exec -it consul-server-0 -- /bin/sh
+  $ export CONSUL_HTTP_TOKEN=<INSERT BOOTSTRAP ACL TOKEN> # export ACL token if ACLs are enabled
+  $ curl "localhost:8500/v1/health/connect/backend?peer=cluster-02"
+  ```
```

Comment:
```suggestion
    ```shell-session
    $ kubectl exec -it consul-server-0 -- /bin/sh
    $ export CONSUL_HTTP_TOKEN=<INSERT BOOTSTRAP ACL TOKEN> # export ACL token if ACLs are enabled
    $ curl "localhost:8500/v1/health/connect/backend?peer=cluster-02"
    ```
```
I think you need to indent this block a bit more, but I can't tell by how much.